internal/controller/http/restrouter.go

```diff
@@ -59,10 +59,10 @@ func (c *RestController) InitRestRoutes() {
 	authenticationHook := handlers.AutoConfigAuthenticationFunc(secretProvider, c.lc)
 
 	// discovery
-	c.addReservedRoute(common.ApiDiscoveryRoute, WrapHandler(c.Discovery), http.MethodPost, authenticationHook)
+	c.addReservedRoute(common.ApiDiscoveryRoute, c.Discovery, http.MethodPost, authenticationHook)
 	// device command
-	c.addReservedRoute(common.ApiDeviceNameCommandNameRoute, WrapHandler(c.GetCommand), http.MethodGet, authenticationHook)
-	c.addReservedRoute(common.ApiDeviceNameCommandNameRoute, WrapHandler(c.SetCommand), http.MethodPut, authenticationHook)
+	c.addReservedRoute(common.ApiDeviceNameCommandNameEchoRoute, c.GetCommand, http.MethodGet, authenticationHook)
+	c.addReservedRoute(common.ApiDeviceNameCommandNameEchoRoute, c.SetCommand, http.MethodPut, authenticationHook)
```

Comment:
I don't agree with how this was handled with all new constants.

Reply:
I think that we need to do it this way if we don't want to break the build.  We can proceed and then go back to the old constant names if we decide we are attached to the old names.  Otherwise, I don't see we need to keep the old names around once everyone is converted.

Reply:
Technically, removing the old constants IS a breaking interface change.  Maybe we should go ahead and plan to revert to the old names once everyone is cut over.